src/deployment.js

```diff
@@ -109,7 +109,7 @@ class Deployment {
       const statusUrl =
         this.deploymentInfo != null
           ? this.deploymentInfo['status_url']
-          : `${this.githubApiUrl}/repos/${this.repositoryNwo}/pages/deployment/status/${process.env['GITHUB_SHA']}`
+          : `${this.githubApiUrl}/repos/${this.repositoryNwo}/pages/deployment/status/${this.buildVersion}`
```

Comment:
We don't use other environment variable in this file so normalizing to what's in the context. Did the same for `cancel` below.